src/UI/Root/Components/Header/Header.tsx

```diff
@@ -22,17 +22,35 @@ import { DocumentationLinks } from "./Actions/DocumentationLinks";
 import { StatusButton } from "./Actions/StatusButton";
 import { EnvSelectorWithProvider } from "./EnvSelector";
 
+/**
+ * Properties for the Header component.
+ *
+ * @interface
+ * @property {boolean} noEnv - A flag indicating whether there is no environment selected.
+ * @property {function} onNotificationsToggle - A function to be called when the notifications badge is clicked.
+ */
 interface Props {
   noEnv: boolean;
   onNotificationsToggle(): void;
 }
 
+/**
+ * Header component of the application.
+ *
+ * This component is responsible for rendering the header of the application.
+ *
+ * @component
+ * @param {Props} props - The properties that define the behavior of the header.
```

Comment:
there's an 
`@props `
and `@prop `
that you can use
we might want to make a decision on this to have everything consistent